admin/templates/configuration/etc/log4cxx.properties

```diff
@@ -5,6 +5,6 @@
 log4j.rootLogger=DEBUG, QLogger
 log4j.appender.QLogger=org.apache.log4j.ConsoleAppender
 log4j.appender.QLogger.layout=org.apache.log4j.PatternLayout
-log4j.appender.QLogger.layout.ConversionPattern=%d{MMdd HH:mm:ss.SSS} [%t] %-5p %c{2} (%F:%L) - %m%n
+log4j.appender.QLogger.layout.ConversionPattern=%d{MMdd HH:mm:ss.SSS} [PID:%X{PID}, %t] %-5p %c{2} (%F:%L) - %m%n
```

Comment:
Why does qserv need PID in the log? I thought it was single-process application.